metacat-connector-druid/src/main/java/com.netflix.metacat.connector.druid/IMetacatDruidClient.java

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2017 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ */
+
+package com.netflix.metacat.connector.druid;
+
+import java.util.List;
+import java.util.Map;
+
+
+/**
+ * Druid Client.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+public interface IMetacatDruidClient {
```

Comment:
Do we have a consistent naming convention in Metacat for Interfaces? I feel like in some places we don't have the `I` and some places we do. We should probably be consistent one way or the other.

Reply:
Just to add onto this I'm personally a fan of the interface being the name of whatever you want e.g. `DatabaseService` instead of `IDatabaseService` cause then the code becomes more readable when you're using the interfaces instead of the concrete classes. Then you can have something like `DatabaseServiceImpl`